locations/spiders/mccoys.py

```diff
@@ -0,0 +1,65 @@
+import datetime
+import scrapy
+import re
+
+from locations.items import GeojsonPointItem
+from locations.hours import OpeningHours
+
+
+class McCoysSpider(scrapy.Spider):
+    name = "mccoys"
+    allowed_domains = ["www.mccoys.com"]
+    download_delay = 0.5
+    start_urls = (
+        'https://www.mccoys.com/stores',
+    )
+
+    def parse_hours(self, hours):
+
+        opening_hours = OpeningHours()
+        for i in range(0, len(hours), 2):
+            day = hours[i]
+            open_time, close_time = hours[i+1].split('-')
+            opening_hours.add_range(day=day[:2],
+                                    open_time=datetime.datetime.strptime(open_time.strip(), '%I:%M %p').strftime('%H:%M'),
```

Comment:
I recently made a change to `add_range()` that does the `strptime()`/`strftime()` dance for you if you provide the `strptime` format as a third parameter to `add_range()`. Can you give that a try here?

Reply:
Ah, that is great!  I just made the change